src/js/pages/system/UnitsHealthDetail.js

```diff
@@ -47,6 +55,24 @@ class UnitsHealthDetail extends mixin(StoreMixin) {
     UnitHealthStore.fetchUnitNodes(this.props.params.unitID);
   }
 
+  onUnitHealthStoreUnitSuccess() {
+    let isLoading = this.state.isLoading & ~FLAGS.UNIT_LOADED;
```

Comment:
Could we create some wrapper methods to make it more descriptive? Something like: `getFlagValue` that does return a `bool`?